### PR TITLE
Finish some tasks to set up the repository.

### PR DESCRIPTION
**Pull request by MaHaWo**
- [x] make config dummy to ensure it works 
- [x] use pyproject.toml instead of setup.py, requirements.txt 
- [x] change testing to unittest 
- [x] add some additional issue templates 
- [x] update readme, pyproject.toml, gitignore
- [x] add a dummy test to be filled in later 
- [x] add model files using lsf -> for tests

### CODE REVIEW

**Comment by MaHaWo** [comment]
@iulusoy would you like to have a look at this? not much happened - just added some files (model files for testing, etc, see description...). Haven't worked a lot with setting up python projects...

**Comment by MaHaWo** [comment]
made a mistake: added on accident a binary file and a bunch humongous csv files for the google-perch model, which should probably live in git lsf... should this be done separately?

**Review by iulusoy — COMMENTED** [review]
Looks good, I made some small mostly cosmetic changes.
We need dependencies for developers too, so I added `requirements-dev.txt`.
I would strongly suggest to use `pytest` as testing framework, as it is highly flexible and allows use of many pre-defined fixtures.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -31,9 +31,9 @@ jobs:
...
-        pip install -r requirements.txt
-    - name: Run pytest
+        python3 -m pip install --upgrade pip
+        python3 -m pip install .
+    - name: Run unittest
       run: |
-        cd src/iSparrow
-        python -m unittest
+        cd .
+        python3 -m unittest discover -s ./tests/ -p "*_test.py"
```
[the remark this answers is not in the document]
Comment (MaHaWo) [reply .github/workflows/main.yml:39]: personal preference only. used unittest in the past, pytest not so much.

[the remark this answers is not in the document]
Comment (iulusoy) [reply .github/workflows/main.yml:39]: I would strongly push for pytest, I can also set up the initial structure of the tests if you like. 
(Would do that in a separate PR, after this one is merged and if you tell me which functions/methods are relevant)

[the remark this answers is not in the document]
Comment (MaHaWo) [reply .github/workflows/main.yml]: re-added pytest.

File: main.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+import birdnetlib as bnl
+
+from src.iSparrow import analyzer as ana
+from src.iSparrow import recording as rec
+from src.iSparrow import preprocessor as pre
+
+# home = Path.home()/ Path('Development/iSparrow')
+home = Path.home() / Path(
+    "Library/CloudStorage/SeaDrive-HaraldMack(heibox.uni-heidelberg.de)/Meine Bibliotheken/iSparrow"
```
[the remark this answers is not in the document]
Comment (MaHaWo) [reply main.py]: removed the file for now, it's experimental only and shouldn't be there actually...

File: main.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+)
+with open(home / "tests/test.yml", "r") as file:
```
[the remark this answers is not in the document]
Comment (MaHaWo) [reply main.py]: removed the file for now, it's experimental only and shouldn't be there actually...

**Comment by MaHaWo** [comment]
@iulusoy 

- re-added pytest 
- resolved an issue with the gitignore (forgotten "!") 
- removed the experimental "main.py"

**Pull request merged**